Formula/neomutt.rb

```diff
@@ -0,0 +1,34 @@
+class Neomutt < Formula
+  desc "E-mail reader with support for Notmuch, NNTP and much more"
+  homepage "https://www.neomutt.org/"
+  url "https://github.com/neomutt/neomutt/archive/neomutt-20171215.tar.gz"
+  sha256 "7fb76e99a9f23715ad772ad8f7008c6e2db05eed344817055176c76dbd60c1b5"
+  head "https://github.com/neomutt/neomutt.git"
+
+  depends_on "docbook-xsl" => :build
+  depends_on "gettext" => :build
```

Comment:
This is still a run time requirement not `=> :build`. The test is not failing because it's being brought in indirectly as a runtime requirement by `gnupg` and `glib`.

Reply:
OK, I'll move it into runtime dependencies.